bin/git-ca

```diff
@@ -0,0 +1 @@
+git commit --amend -v --date=$(date +%Y-%m-%dT%H:%M:%S)
```

Comment:
Somebody also commented that `-R` doesn't work in OSX. Saying an alternative is:
~~~
git commit --amend --date="$(date +%s)"
~~~